src/main/java/com/github/junit5docker/DockerExtension.java

```diff
@@ -7,7 +7,7 @@
 import java.util.HashMap;
 import java.util.concurrent.*;
 
-public class DockerExtension implements BeforeAllCallback, AfterAllCallback {
+class DockerExtension implements BeforeAllCallback, AfterAllCallback {
```

Comment:
The DockerExtension does not have to be public so we can avoid useless documentation.